README.md

```diff
@@ -17,12 +17,12 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
+[X] Strings
+[X] Booleans
 [] Undefined
 [] NaN
-[] Integers
-[] Arrays
+[X] Integers
+[X] Arrays
```

Comment:
`undefined` actually *is* a data type, and instead of `Integer` Javascript uses `Number`. `Array` is technically not a data type -- it's an object -- but it's so fundamental that it's basically considered one. May be good to remember for interviews, though.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures